src/Tribe/RSVP/Service_Provider.php

```diff
@@ -48,6 +49,9 @@ private function register_early_access() {
 		add_filter( 'tribe_events_tickets_template_tickets/rsvp.php', [
 			$this->container->make( Template::class ),
 			'override_template'
+		add_action( 'admin_init', [
```

Comment:
* :no_entry_sign: PHP syntax error: syntax error, unexpected 'add_action' (T_STRING), expecting ']'
* :no_entry_sign: Opening parenthesis of a multi-line function call must be the last content on the line
* :no_entry_sign: Only one argument is allowed per line in a multi-line function call

<sub>hash: b840f44640ec9a66fbd53375d29fc224</sub>